packages/core/src/form-field/FormFieldLabel.tsx

```diff
@@ -10,11 +10,16 @@ import formFieldLabelCss from "./FormFieldLabel.css";
 
 const withBaseName = makePrefixer("saltFormFieldLabel");
 
+type NecessityType = "required" | "optional" | "*";
```

Comment:
Are we promoting "*"? I thought there's a reason we use words, maybe a11y reason

How are users going to support non-english labels?